pdfannots/types.py

```diff
@@ -297,7 +297,8 @@ def __init__(
             rect: typ.Optional[BoxCoords] = None,
             contents: typ.Optional[str] = None,
             author: typ.Optional[str] = None,
-            created: typ.Optional[datetime.datetime] = None):
+            created: typ.Optional[datetime.datetime] = None,
+            color: typ.Optional[RGB] = None): # RGB color expressed as a 3-tuple of floats in [0, 1]
```

Comment:
Please document the color attribute in the block comment (line 279 of this file).